sos/sosreport.py

```diff
@@ -1296,6 +1308,17 @@ def final_work(self):
                 except (OSError, IOError):
                     print(_("Error moving checksum file: %s" % archive_hash))
 
+        if self.opts.upload or self.opts.upload_url:
```

Comment:
I'd prefer if this goes after display_results.  That way they have more info - like how big the file is.

Reply:
Fair point, I'll get that moved.